src/components/layout.css

```
@@ -528,3 +899,29 @@ main {
   top: 2000px;
   width: 3px;
 }
+
+/* TOC */
+details,
+.toc {
+  cursor: pointer;
+}
+.toc h6 {
+  display: inline-block !important;
```

1. Does it not work without the `!important`?

2. Adding `margin: 0` fixes the layout problem caught by @LaRuaNa.

```suggestion
  display: inline-block;
  margin: 0;
```

---

Also removed the important tag since I had already moved the CSS at the end.